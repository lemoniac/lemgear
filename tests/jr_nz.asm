	LD A, 10
l:
	DEC A
	CP 0
	JR NZ, l
	
	LD B, 2
	
	HALT
