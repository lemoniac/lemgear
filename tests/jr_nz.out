AF : 0000
BC : 0200
